LICENSE.md

```diff
@@ -0,0 +1,22 @@
+Skript is licensed under the [GNU General Public License (Version 3)](LICENSE).
+
+However, to foster a better open-source community, and in the interest of further opening this project in the future,
+we ask contributors to release their code under the [MIT License](MIT.txt).
+Since it is less restrictive than GPLv3, it provides us and other developers
+with greater flexibility when making use of your code.
+
+To learn more about this license, you can visit [choosealicense.com](https://choosealicense.com/licenses/mit/).
+
+If you wish to release your contributions to this repository under the [MIT License](MIT.txt),
+please add your name below in your pull request.
+
+When adding your name, please follow the example format showcased below:
```

Comment:
I think this file should be called something different so it doesn't get confused with the actual licence.

Reply:
Do you think `LICENSING` would be too similar?